<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateEditorWidget</class>
 <widget class="QFrame" name="TemplateEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <widget class="QTextEdit" name="templateTextEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>

      </widget>
    </item>
    <item>
      <widget class="QPushButton" name="templateTextSave">
        <property name="text">
         <string> Save Changes </string>
        </property>
      </widget>
    </item>
    <item>
      <widget class="QPushButton" name="templateTextSaveAs">
        <property name="text">
         <string> Save Changes As... </string>
        </property>
      </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
